mmseg/datasets/transforms/transforms.py

```diff
@@ -1226,3 +1226,74 @@ def __repr__(self):
         repr_str += f'edge_width={self.edge_width}, '
         repr_str += f'ignore_index={self.ignore_index})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class RandomGamma(BaseTransform):
+    """Using random gamma correction to process the image.
+
+    Modified from https://docs.monai.io/en/0.3.0/_modules/monai/transforms/intensity/array.html#RandAdjustContrast
+
+    Required Keys:
+    
+    - img
+
+    Modified Keys:
+
+    - img
+
+    Args:
+        prob (float): RandomGamma probability.
+        gamma_range: Range of gamma values.
+    """
+
+    def __init__(self, prob: float,
+                 gamma_range: tuple[float] = (0.5, 4.5)) -> None:
+        super().__init__()
+        assert 0 <= prob and prob <= 1
+        assert isinstance(gamma_range, tuple)
```

Comment:
do we need to add a requirement about gamma>0.5?

Reply:
Yeah, I have added such an assertion in the latest commit.